### DISCUSSION TITLE
Noise Uncertainty Quantification (NUQ)

### DISCUSSION THREAD

**Discussion by JihunKwon** [dipy/dipy#2720] [3. Community Lounge]
Hi, I have a question about NUQ (Noise Uncertainty Quantification) presented at DIPY workshop 2022. Is this functionality already implemented in DIPY? I am interested in using it to my project.

**Comment by ShreyasFadnavis** [comment]
Hi @JihunKwon ! The code is not merged in DIPY yet... but I can share it with you personally. Is there an email I can share it with you on?

**Reply by JihunKwon** [reply]
Hi @ShreyasFadnavis , thank you for your quick reply. Could you please use this adress: jihun.k<at>frontier.hokudai.ac.jp ? Please insert @ to <at>.

**Reply by ShreyasFadnavis** [reply]
> Hi @ShreyasFadnavis , thank you for your quick reply. Could you please use this adress: jihun.k<at>frontier.hokudai.ac.jp ? Please insert @ to <at>.
> 
>  

Got it! Thanks👍🏽

**Reply by JihunKwon** [reply]
Hi @ShreyasFadnavis , sorry, I realized that my email was not correct and your email was not received so far. Could you please use this email adress: jihun.kATfrontier.hokudai.ac.jp (replace AT with @).

Just one more question. I work in industry and my instention is to use NUQ as an objective DWI image quality assessment tool. Does NUQ allows the use in industry?

**Comment by skoudoro** [comment]
is there an ETA for NUQ in DIPY @ShreyasFadnavis? Just curiosity, if not, that's completly fine!

**Reply by ShreyasFadnavis** [reply]
I aim to get it in before the workshop @skoudoro !

**Comment by njhunsak** [comment]
Just checking that the NUQ code never got implemented with DIPY? I don't see it anywhere in the documentation or on the DIPY site. Thanks.

**Reply by skoudoro** [reply]
I confirm, it has not been added to DIPY. 

Feel free to contact the author @ShreyasFadnavis  to get the code.

I will do the same from my side and see if we can add this task.
